### DISCUSSION TITLE
yolov8n.tflite

### DISCUSSION THREAD

**Discussion by gpsamt** [ultralytics/ultralytics#1409] [General]
Hello. I'm studying yolo8 to run on an esp32. I'm following https://docs.ultralytics.com/#ultralytics-yolov8 . I didn't find yolov8n.tflite . Where is this file? Can anybody help me? Thanks!!

**Comment by dh031200** [comment]
You can export it yourself.
```bash
yolo export model=path/to/model.pt format=tflite  # export custom trained model
```

Check the page below for more information.
https://docs.ultralytics.com/modes/export/#arguments

**Reply by glenn-jocher** [reply]
@dh031200 hello! To obtain the `yolov8n.tflite` file, you'll need to export the YOLOv8 model to the TensorFlow Lite format yourself. You can do this by using the export mode of YOLOv8 with the appropriate command in your terminal. Please refer to the Export section of our documentation for detailed instructions on the arguments and usage. The process is straightforward and allows you to convert the model to a format compatible with your ESP32 device. If you have any further questions or need assistance, feel free to reach out. Happy coding! 😊
